website/src/components/edit/edit.tsx

```diff
@@ -45,7 +45,7 @@ const Edit: React.FC<{
   signedInEmail: string | boolean;
 }> = ({ currentData, currentKey, editing, signedInEmail }) => {
   const dispatch = useDispatch();
-  const history = useHistory();
+  const navigate = useNavigate();
```

Comment:
https://reactrouter.com/docs/en/v6/upgrading/v5#use-usenavigate-instead-of-usehistory